Source/WebKit/WPEPlatform/wpe/drm/WPEDRM.cpp

```diff
@@ -81,10 +81,6 @@ std::unique_ptr<Connector> Connector::create(int fd, drmModeConnector* connector
     if (!properties)
         return nullptr;
 
-    auto dpms = drmPropertyForName(fd, properties.get(), "DPMS");
```

Comment:
This is a legacy property: https://drmdb.emersion.fr/properties/3233857728/DPMS

Reply:
I dont think there is any advantage of checking for this during the initialization.

Reply:
Fair enough, but let's split this use https://bugs.webkit.org/show_bug.cgi?id=268533 for this change only

Reply:
Implemented in https://github.com/WebKit/WebKit/pull/23654